### PR TITLE
BE-1: Add live server

### PR DESCRIPTION
**Pull request by LiskaC**

### CODE REVIEW

**Comment by OleksIva** [comment]
Test

File: server/server.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import http from 'http'
+
+const server = http.createServer((req, res) => {
+
```
Comment (OleksIva) [inline server/server.js:4]: What the hell is this

**Review by OleksIva — APPROVED** [review]
LGTM

File: server/server.js (hunk 1 above)
Comment (LiskaC) [reply server/server.js:4]: dunno

File: server/server.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+})
+
+const PORT = 3000
```
Comment (LiskaC) [inline server/server.js:7]: this is the port i decide to use - what do you think?

**Pull request merged**